tests/app/main/test_services.py

```diff
@@ -301,14 +291,12 @@ def test_should_view_public_service_with_update_message(
 
         res = self.client.get('/suppliers/services/123')
         if not framework_editable_services:
-            assert_equal(res.status_code, 404)
+            assert res.status_code == 404
             return
 
-        assert_equal(res.status_code, 200)
+        assert res.status_code == 200
 
-        assert_true(
-            'Service name 123' in res.get_data(as_text=True)
-        )
+        assert'Service name 123' in res.get_data(as_text=True)
```

Comment:
I'm amazed PEP8 doesn't choke on this, but could do with a space after `assert` here.